src/stylesheets/components/_card.scss

```diff
@@ -1,29 +1,43 @@
 .sds-card {
-  @include u-margin-y(5);
-  @include u-padding-top(2);
-  @include u-padding-x('205');
-  @include u-padding-bottom('205');
+  @include u-margin-y(2);
+  // @include u-padding-top(2);
+  // @include u-padding-x("205");
+  // @include u-padding-bottom("205");
   border-radius: 4px;
   background-color: white;
-  box-shadow: 0 10px 25px 5px rgba(0, 0, 0, 0.15);
+  box-shadow: 0 6px 12px 0px rgba(0, 0, 0, 0.1);
```

Comment:
Let's use a value from the USWDS library if possible, work with the designer to adjust if needed.

https://designsystem.digital.gov/utilities/shadow/